src/lib/sci-o/index.ts

```diff
@@ -0,0 +1,44 @@
+import {IImpactModelInterface} from '../interfaces';
+import {KeyValuePair} from '../../types/boavizta';
+
+export class SciOModel implements IImpactModelInterface {
+  authParams: object | undefined = undefined;
+  staticParams: object | undefined;
+  name: string | undefined;
+  authenticate(authParams: object): void {
+    this.authParams = authParams;
+  }
+
+  calculate(observations: object | object[] | undefined): Promise<object[]> {
+    if (!Array.isArray(observations)) {
+      throw new Error('observations should be an array');
+    }
+    observations.map((observation: KeyValuePair) => {
+      if ('grid-ci' in observation && 'energy' in observation) {
+        const grid_ci = parseFloat(observation['grid-ci']);
+        const energy = parseFloat(observation['energy']);
+        observation['operational-emissions'] = grid_ci * energy;
```

Comment:
Does operation-emissions exist in the units.yml?

Reply:
It was mentioned in the issue.

Reply:
I mean as you develop models, the units.yml needs also be updated in parallel. Will add it to the checklist to be made clear.